source/about/frequently-asked-questions.rst

```diff
@@ -444,11 +444,11 @@ Yes, the Guest Account feature allows organizations to collaborate with external
 Can I use Mattermost for customer service?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You are welcome to use the open source Mattermost Team Edition and its extensive API library to build your own solution.
+You are welcome to use the open source Mattermost Team Edition and its extensive API library to build your solution.
```

Comment:
```suggestion
You are welcome to use the open source Mattermost Team Edition and its extensive API library to build your own solution.
```

Reply:
In this case, "own" implies ownership which would also include aspects such as support. I think it's important to specify that.

Reply:
Okay but I thought it was giving a redundancy.